src/zFooter/Footer.module.scss

```diff
@@ -0,0 +1,29 @@
+.redRibbon{
+
+  background-color: #cb463e;
```

Comment:
Rather than inlining colors, borders, fonts, etc., put all of them in a `theme` file, and then reference your theme colors by name. This makes updating these values both easier, and more reliable.

Reply:
Done 🐤 😄